gen/nvml/init.go

```diff
@@ -33,12 +33,14 @@ var nvml *dl.DynamicLibrary
 func Init() Return {
 	lib := dl.New(nvmlLibraryName, nvmlLibraryLoadFlags)
 	if lib == nil {
-		panic(fmt.Sprintf("error instantiating DynamicLibrary for %s", nvmlLibraryName))
+		log.Printf("error instantiating DynamicLibrary for %s\n", nvmlLibraryName)
```

Comment:
Question: What is the benefit of using `log.Printf` over `fmt.Printf`?

Reply:
It would add a datetime prefix.

Reply:
I'm OK using log over fmt but we should do it everywhere then (i.e. also in the `dl` library).